src/helm/benchmark/metrics/bhasa_metrics.py

```diff
@@ -0,0 +1,192 @@
+import re
+import string
+from dataclasses import replace
+from typing import Callable, Dict, List
+from collections import Counter
+
+from pythainlp.tokenize import word_tokenize
+from sacrebleu.metrics import CHRF
+
+from helm.benchmark.adaptation.adapter_spec import AdapterSpec
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.metrics.metric import Metric
+from helm.benchmark.metrics.metric_name import MetricName
+from helm.benchmark.metrics.metric_service import MetricService
+from helm.benchmark.metrics.statistic import Stat
+
+class BhasaMachineTranslationMetric(Metric):
+    """Machine Translation Metrics
+
+    This class computes the following standard machine translation metrics
+
+    1. chr_f_plus_plus (ChrF++)
+
+    @inproceedings{popovic-2015-chrf,
+        title = "chr{F}: character n-gram {F}-score for automatic {MT} evaluation",
+        author = "Popovi{\'c}, Maja",
+        editor = "Bojar, Ond{\v{r}}ej  and
+        Chatterjee, Rajan  and
+        Federmann, Christian  and
+        Haddow, Barry  and
+        Hokamp, Chris  and
+        Huck, Matthias  and
+        Logacheva, Varvara  and
+        Pecina, Pavel",
+        booktitle = "Proceedings of the Tenth Workshop on Statistical Machine Translation",
+        month = sep,
+        year = "2015",
+        address = "Lisbon, Portugal",
+        publisher = "Association for Computational Linguistics",
+        url = "https://aclanthology.org/W15-3049",
+        doi = "10.18653/v1/W15-3049",
+        pages = "392--395",
+        github = "https://github.com/mjpost/sacrebleu",
+    }
+    """
+
+    def __init__(self):
+        self.chrf_scorer = CHRF(word_order=2)
+
+    def chr_f_plus_plus(self, refs: List[str], pred: str) -> Dict[str, float]:
+        metrics: Dict[str, float] = {}
+        metrics['chr_f_plus_plus'] = self.chrf_scorer.sentence_score(pred, refs).score
+        return metrics
+
+    def remove_braces(self, text: str) -> str:
```

Comment:
@raileymontalan There's no need for this function actually. If I'm not wrong, this is something that was found in the `SummarizationMetric` (https://github.com/stanford-crfm/helm/blob/main/src/helm/benchmark/metrics/summarization_metrics.py) because they used to use braces to contain the summary, and they used `}` as a stop token. But it seems like the separator between few-shot instances for summarization has been changed to `###`. Since we do not actually include any instructions in our prompt to provide the translation within braces, there is no need for this cleaning step.

@yifanmai This also answers your comment from (https://github.com/stanford-crfm/helm/pull/2694#discussion_r1629822826). 
We might want to remove this from `SummarizationMetric` as well? Or is that perhaps there for backward compatibility?

Reply:
Let's leave it there for backwards compatibility for now.